### PR TITLE
Adding contribution file

### PR DESCRIPTION
**Pull request by OfirBachar**
Adding contribution.md file

### CODE REVIEW

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by idoyek — APPROVED** [review]
LGTM

File: contribution.md
Comment (Yarboa) [inline contribution.md:62]: No pytest here, at this status

File: contribution.md
Comment (Yarboa) [inline contribution.md:44]: Template is not important

You should describe what should be in the description

File: contribution.md
Comment (Yarboa) [reply contribution.md:41]: Consider this, (please make it more friendly)

Create PR
1) Clear subject
2) Clear description, 
3) AddResolve #Issue
4) Assign PR to yourself

File: contribution.md
Comment (Yarboa) [inline contribution.md:42]: Consider the foolowing
move to -> \#\# Review Process

And describe the minimum for fast attention 
1) Verify Githab actions tests status = PASSED
2) Select Reviewers under Reviewers section

Note: at least 2 LGTM, Approves needed for Core reviewers to review

File: contribution.md
Comment (Yarboa) [inline contribution.md:52]: Please also explain the difference between Reviewers and CoreReviers

**Review by Yarboa — CHANGES_REQUESTED** [review]

File: contribution.md
Comment (OfirBachar) [inline contribution.md:1]: Deleting unnecessary lines/spaces

**Review by patrisiakaplun — APPROVED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by OfirBachar — COMMENTED** [review]
LGTM

**Review by Yarboa — COMMENTED** [review]
Once you changed 
1. Reviewers
2.  The Maintainers 

Please use it in the whole doc.
Please replace mentors with something like this 

https://github.com/redhat-beyond/Pro4U/pull/16#discussion_r1145285200

**Review by idosinger98 — APPROVED** [review]
LGTM

File: contribution.md
Comment (Yarboa) [inline contribution.md]: I still see it, Please change Core reviewers to maintainers

File: contribution.md
Comment (Yarboa) [reply contribution.md:57]: consider change to this 
are project maintainers
They are final decision makers on which portions of source code go into a build or release

I think this should move to before "Create PR"

**Review by idosinger98 — APPROVED** [review]
LGTM

**Review by OfirBachar — COMMENTED** [review]
LGTM

**Review by TalReinfeld — APPROVED** [review]
LGTM

File: contribution.md
Comment (brochwerger) [inline contribution.md:33]: Keep terminology consistent - stick to "Reviewers" and "Maintainers", no "fast reviewers")

File: contribution.md
Comment (brochwerger) [inline contribution.md:54]: not clear - did you mean two LGTM and two approvals from Reviewers before Maintaners review ?

**Review by TalReinfeld — APPROVED** [review]
LGTM

**Review by idosinger98 — APPROVED** [review]
LGTM

File: contribution.md
Comment (guybeckenstein) [inline contribution.md:21]: Note that it is also possible to enter the website using http://127.0.0.1:8000/, but I don't find this note mandatory

**Review by guybeckenstein — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]

**Pull request merged**
